### ISSUE TITLE
SQL Zoo: SQL Zoo link is broken or not working

### ISSUE THREAD

**Issue by Hossam-Allam** [TheOdinProject/curriculum#29163]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The sql zoo link in the assignment leads to a blank page

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/databases-sql-zoo

**Comment by Captainmango** [comment]
I have a solution for this and will raise a PR shortly.

The cause is that the `sqlzoo.net` URL forces a redirect which Edge and Firefox (the browsers I tested) do not seem to follow. I was able to get the page to render in Chrome, but only after clearing my local cache.

sqlzoo has a permanent link on their page that forces the latest revision of the site. However, this would become stale very quickly. Instead, it looks like they have an option of sending a URL param to turn the redirect behaviour off.

`https://sqlzoo.net/w/index.php?title=SQL_Tutorial&redirect=no` I tried this in all the browsers I tested and it works as expected.

**Comment by Captainmango** [comment]
PR that addresses this issue here #29167
